response.go

```diff
@@ -128,6 +187,25 @@ func (r *Response) Cookie(cookie *http.Cookie) {
 	http.SetCookie(r.responseWriter, cookie)
 }
 
+// Flush sends any buffered data to the client if the underlying
+// writer implements `goyave.Flusher`.
+//
+// If the response headers have not been written already, `PreWrite()` will
+// be called with an empty byte slice.
+func (r *Response) Flush() {
+	if !r.wroteHeader {
+		r.PreWrite([]byte{})
```

Comment:
I'm a bit surprised by this line, because it will call prewrite, that will indeed send the header, but I feel strange to PreWrite during flush

Shouldn't you simply call WriteHeader ?
```suggestion
		r.WriteHeader(r.status)
```

Reply:
A flush will trigger a write so we want to trigger `PreWrite` as well. It's only relevant if you call `Flush` without having written anything (no prior call to `Write`).